tests/phpunit/tests/rest-api/rest-block-type-controller.php

```diff
@@ -690,6 +699,7 @@ protected function check_block_type_object( $block_type, $data, $links ) {
 			'parent',
 			'provides_context',
 			'uses_context',
+			'block_hooks',
 			'supports',
 			'styles',
 			'textdomain',
```

Comment:
@gziolo BTW is this list missing `selectors`?

Reply:
Yes, it's likely an issue here.

Reply:
Although, it's present in the response:

https://github.com/WordPress/wordpress-develop/blob/f3a04cb121bcd4e4df61aeb08837a3182c54cb28/src/wp-includes/rest-api/endpoints/class-wp-rest-block-types-controller.php#L525-L532

I see it only in the dev docs:

https://developer.wordpress.org/rest-api/reference/block-types/

Reply:
I think we should file a separate PR for this, mostly for visibility.

Reply:
I can locate the old ticket and commit with a reference to it as a follow-up in the next hour.

Reply:
@gziolo That'd be great, thank you! I think it's https://core.trac.wordpress.org/ticket/57585 and https://core.trac.wordpress.org/changeset/55673 😊

Reply:
It was addressed in https://core.trac.wordpress.org/changeset/56588.